### PR TITLE
Desktop 2fa

### PR DESCRIPTION
**Pull request by usha-mandya**
PR to introduce Desktop 2 FA updates.

Ref: https://docker.atlassian.net/browse/DESKTOP-1864

### CODE REVIEW

File: docker-for-mac/index.md [hunk 1]
```diff
@@ -494,13 +494,31 @@ For information about providing feedback on the documentation or update it yours
...
+
+You must enable two-factor authentication in Docker Hub before signing into your Docker Hub account through Docker Desktop. For instructions, see [Enable two-factor authentication for Docker Hub](/docker-hub/2fa/).
+
+After you have enabled two-factor authentication:
+
+1. Go to the Docker Desktop menu and then select **Sign in / Create Docker ID**.
+
+2. Enter your Docker ID and password and click **Sign in**.
+
+3. After you have successfully signed in, Docker Desktop prompts you to enter the authentication code. Enter the six digit code from your phone and then click **Verify**.
```
Comment (traci-morrison) [inline docker-for-mac/index.md]: s/six digit code/six-digit code

File: docker-for-windows/index.md [hunk 2]
```diff
@@ -536,6 +536,24 @@ For more information, refer to the following [Docker Hub topics](/docker-hub/ind
...
+
+You must enable two-factor authentication in Docker Hub before signing into your Docker Hub account through Docker Desktop. For instructions, see [Enable two-factor authentication for Docker Hub](/docker-hub/2fa/).
+
+After you have enabled two-factor authentication:
+
+1. Go to the Docker Desktop menu and then select **Sign in / Create Docker ID**.
+
+2. Enter your Docker ID and password and click **Sign in**.
+
+3. After you have successfully signed in, Docker Desktop prompts you to enter the authentication code. Enter the six digit code from your phone and then click **Verify**.
```
Comment (traci-morrison) [inline docker-for-windows/index.md]: s/six digit code/six-digit code

File: docker-for-mac/index.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-mac/index.md]: Thanks. Fixed.

File: docker-for-windows/index.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/index.md]: Thanks. Fixed.

**Pull request merged**